QuickOSM/test/test_query_factory.py

```diff
@@ -600,7 +600,22 @@ def test_make(self):
             '<recurse type="down"/>\n    </union>\n    '
             '<print mode="body"/>\n</osm-script>\n'
         )
-        self.assertEqual(query.make(), expected)
+        self.assertEqual(query.make(False), expected)
+
+        query = QueryFactory(
```

Comment:
As it seems we will keep `generate_oql` and `generate_xml`, you need to add a few more test case to have a good code coverage (loop, many places, different kind of queries). Similar to what is done with OQL

I think the XML in Overpass will slowly disappear or become unknown.
But we need to keep a good code coverage about the QueryFactory with OQL.

ABove, there is a test 

` def test_query(q, xml, xml_with_template, human_label):`

Why not giving a "oql" and then we check all queries against xml and oql at the same time ?